test/nodes/test_connector.py

```diff
@@ -259,3 +258,12 @@ def test_crawler_depth_2_multiple_urls(test_url, tmp_path):
     assert content_in_results(crawler, test_url + "/page1_subpage1.html", paths)
     assert content_in_results(crawler, test_url + "/page1_subpage2.html", paths)
     assert content_in_results(crawler, test_url + "/page2_subpage1.html", paths)
+
+
+def test_crawler_custom_webdriver(tmp_path):
+    tmp_dir = tmp_path
+
+    webdriver = Chrome()
+    crawler = Crawler(output_dir=tmp_dir, file_path_meta_field_name="file_path", webdriver=webdriver)
```

Comment:
```suggestion
@pytest.mark.unit
def test_crawler_custom_webdriver():
    webdriver = Chrome()
    crawler = Crawler(webdriver=webdriver)
```

My impression is that you can simplify the test in this way...